src/arviz_plots/plots/compareplot.py

```diff
@@ -0,0 +1,137 @@
+"""Compare plot code."""
+from importlib import import_module
+
+from arviz_base import rcParams
+
+
+def plot_compare(
+    cmp_df,
+    color="black",
+    similar_band=True,
+    relative_scale=False,
+    figsize=None,
+    target=None,
+    backend=None,
```

Comment:
I would maintain `plot_kwargs` because it still makes sense, the rest aes_map, stats_kwargs make no sense so I agree they should not be here.